src/adagrams.js

```diff
@@ -1,15 +1,121 @@
+const LETTER_POOL = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+};
 export const drawLetters = () => {
-  // Implement this method for wave 1
+
+  let drawedLetters = [];
+  const numAllowedinHand = 10;
+  let allLetters = [];
+
+  for (const [key, value] of Object.entries(LETTER_POOL)){
+    for (let i = 0; i < value; i++){
+      allLetters.push(key);
+    };
+  };
+
+  while (drawedLetters.length < numAllowedinHand){
+    let randomNumber = Math.floor(Math.random() * allLetters.length);
```

Comment:
`randomLetter` is initialized and assigned a value each time the loop runs. The variable is not reassigned in the while loop so we should use `const`
```suggestion
    const randomNumber = Math.floor(Math.random() * allLetters.length);
```